apps/api/src/app/workflows-v1-deprecated/usecases/get-active-integrations-status/get-active-integrations-status.command.ts

```diff
@@ -1,6 +1,10 @@
 import { NotificationTemplateEntity } from '@novu/dal';
 import { EnvironmentWithUserCommand } from '../../../shared/commands/project.command';
 
+/**
+ * @deprecated use commands in /workflows directory
+ */
+
 export class GetActiveIntegrationsStatusCommand extends EnvironmentWithUserCommand {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** remove the extra line, it breaks the typedoc.‏
```suggestion
 */
export class GetActiveIntegrationsStatusCommand extends EnvironmentWithUserCommand {
```

Reply:
done